boards/arduino-uno/Makefile.include

```diff
@@ -0,0 +1,46 @@
+# define the cpu used by the arduino uno board
+export CPU = atmega328p
+
+export TERMPROG = $(RIOTBASE)/dist/tools/pyterm/pyterm
+export TERMFLAGS = -b 9600 -p $(PORT)
+
+#define the flash-tool and default port depending on the host operating system
+OS = $(shell uname)
+ifeq ($(OS),Linux)
+  PORT ?= /dev/ttyUSB0
```

Comment:
Yes, there is some common with mega2560 for example.
Maybe that can be done in a 2nd time ?